src/App.tsx

```diff
@@ -1,24 +1,209 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState, useMemo } from 'react';
 import { UserWarning } from './UserWarning';
+import {
+  getTodos,
+  postTodos,
+  deleteTodo,
+  toggleTodo,
+} from './api/todos';
+import { Todo } from './types/Todo';
+import { FilterStatus } from './types/FilterStatus';
 
-const USER_ID = 0;
+import { Header } from './components/Header/Header';
+import { ListOfTodos } from './components/ListOfTodos/ListOfTodos';
+import { FilterForTodos } from './components/FilterTodos/FilterTodos';
+import {
+  ErrorNotification,
+} from './components/ErrorNotification/ErrorNotification';
+
+const USER_ID = 6984;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState('');
+  const [filter, setFilter] = useState<FilterStatus>(FilterStatus.All);
+  const [filteredTodos, setFilteredTodos] = useState<Todo[]>([]);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [disabledInput, setDisabledInput] = useState(false);
```

Comment:
follow the convention for the boolean variable names

Reply:
done